spec/features/proposal/collection_draft_proposals/collection_draft_proposal_destroy_spec.rb

```diff
@@ -30,19 +30,17 @@
     end
   end
 
-  # This test should be uncommented when the submitted state is added.  It works
-  # without modification, locally, at the time of writing.
-  # context 'when deleting a "Submitted" proposal' do
-  #   before do
-  #     draft = create(:full_collection_draft_proposal, draft_short_name: 'A Deletable Example')
-  #     proposal = CollectionDraftProposal.first
-  #     proposal.proposal_status = 'submitted'
-  #     proposal.save
-  #     visit collection_draft_proposal_path(draft)
-  #   end
+  context 'when deleting a "Submitted" proposal' do
```

Comment:
This should not actually be feasible, right? In that case, I think it should read 'when attempting to delete...'